src/Avalonia.FuncUI.ControlCatalog/Views/Tabs/CanvasDemo.fs

```diff
@@ -24,7 +21,7 @@ module CanvasDemo =
         match msg with
         | SetCount count -> { state with itemCount = count }
            
-    let view (state: State) (dispatch) =
+    let view (_state: State) (_dispatch) =
```

Comment:
is this common practice somehow ? 

It would also be possible to discard both arguments to clarify that they are unused. 

https://sharplab.io/#v2:DYLgZgzgNAJiDUAfA9gBwKYDsAEBlAnhAC7oC2AsAFBWnIwCuw62AgtgLxVXY/ZNHYAhjBjYAFAH0QAS0xEo2KbKIBKbCGzKO3XroAMQA===

Reply:
Yes, it is a common practice. 
For example, take a look at https://github.com/dotnet/fsharp/blob/master/src/fsharp/FSharp.Core/Linq.fs#L168.

I decided not to replace the whole arg name with '_' to save the meaning of arguments.

Reply:
ok